src/pages/login/LoginPage.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useAuthFlag, broadcastLogin } from '../../contexts/AuthFlag';
+import { loginPortal, type LoginRequest } from '../../api/auth';
+import LoginLogo from '../../../assets/loginLogo.png';
+
+export default function LoginPage() {
+  const { setAuthenticated, setUnauthenticated } = useAuthFlag();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const sp = new URLSearchParams(location.search);
+  const rawRedirect = sp.get('redirect');
+  const redirectPath = rawRedirect && rawRedirect.startsWith('/') ? rawRedirect : '/';
+
+  const [portalId, setPortalId] = useState('');
+  const [portalPassword, setPortalPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setErrorMessage(null);
+
+    if (!portalId.trim() || !portalPassword) {
+      setErrorMessage('아이디와 비밀번호를 입력하세요.');
+      return;
+    }
+
+    setIsSubmitting(true);
+    try {
+      const payload: LoginRequest = { portalId, portalPassword };
+      await loginPortal(payload);
+
+      setAuthenticated();
+      broadcastLogin();
+      navigate(redirectPath, { replace: true });
+    } catch (e: any) {
+      setUnauthenticated();
+      if (e?.status === 401) setErrorMessage('아이디 또는 비밀번호가 올바르지 않습니다.');
+      else if (e?.status === 400) setErrorMessage('요청 형식이 올바르지 않습니다.');
+      else setErrorMessage('로그인에 실패했습니다. 잠시 후 다시 시도하세요.');
+    } finally {
+      setIsSubmitting(false);
+    }
+  };
+
+  return (
+    <main className="mx-auto max-w-6xl px-6 pt-16 pb-12 md:pt-24">
+      <section className="overflow-hidden rounded-2xl border shadow-md">
+        <div className="grid md:grid-cols-2">
```

Comment:
<img width="376" height="670" alt="image" src="https://github.com/user-attachments/assets/8a72e155-66dd-4bed-a9a9-32dff4933457" />
<img width="1914" height="906" alt="스크린샷 2025-08-31 010124" src="https://github.com/user-attachments/assets/eaff253f-82fb-4d7f-b973-ab3b592a436d" />

모바일 버전에서는 안내가 안나오도록 하셨군요! 사용자가 보기엔 훨씬 편할 것 같아요!
그런데 안내문의 내용 중에는, "로그인을 위해선 세종대학교 포털에서 개인정보수집 동의가 되어있어야 합니다." 같이 사용자가 알아야 하는 문구가 들어있어서 모바일에서 로그인 할 때도 화면에 나타나야 할 것 같아요.
```suggestion
              <section className="overflow-hidden rounded-2xl border shadow-md">
        <div className="flex flex-col-reverse md:flex-row">
```

안내문의 hidden 속성을 제거해서 모바일 환경에서 로그인 시 안내문을 로그인 화면 아래에 배치하는 건 어떻게 생각하세요??

<img width="376" height="666" alt="image" src="https://github.com/user-attachments/assets/42c896a9-98e9-4532-be76-8293fa5f1446" />

Reply:
저는 개인적으로 아래에 `로그인 안내`를 배치하는 것보다는 구현이 조금 귀찮을 수도 있겠지만 물음표 아이콘? 같은 버튼을 만들어서 안내 모달을 띄우는 방식이 UX적으로 조금 더 좋다고 생각이 듭니다.

모바일을 사용하면서 로그인하는데 스크롤을 내린다는 점이 개인적으로 조금 어울리지 않을 것 같다는 생각이 들어서 의견을 남기게 됐습니다. 참고해주시면 감사하겠습니다!

Reply:
그것도 좋은 생각인 것 같아요!